### PR TITLE
SEAR - 169 Add allegations section to Facility profile complaints history

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/SEAR-169

#### Brief feature description
- Show allegations for each complaint.

#### Does this PR block/effect other stories?
-No

#### Do any commands needs to be run in order to run the project with this PR?
- As the backend is still under development. 
- This is the JSON format for allegations from API.
```  
 'allegations': [{
              'complaint_type_code': '10',
              'complaint_type_description': 'Neglect/Lack of Supervision',
              'allegation': 'Staff Meena Desai marks clients Alexzena Webb and Dizirea Goodwin with a ballpoint pen while they are sleeping.',
              'resolution_type_code': 'U',
              'resolution_type_description': 'Unsubstantiated'
            }]
```

#### Code coverage percent diff as a result of this PR?
-No

### CODE REVIEW

**Comment by sunilb0575** [comment]
jenkins, retest this please

File: app/javascript/facility/facilityComplaints.jsx [hunk 1]
```diff
@@ -12,43 +13,93 @@ export default class FacilityComplaints extends React.Component {
...
-                      <td data-label='priority level'> {complaint.priority_level} </td>
-                      <td data-label='status'> {complaint.status} </td>
-                      <td data-label='approval date'> {complaint.approval_date} </td>
-                    </tr>
-                  )
-                })}
-              </tbody>
-            </table>
+            : <ReactTable
+              id='facility-children-table'
```
Comment (Nasir2712) [inline app/javascript/facility/facilityComplaints.jsx]: shouldn't this be facility-complaints-table?

File: app/javascript/selectors/facilityDataSelectors.js [hunk 2]
```diff
@@ -81,11 +81,21 @@ const getFacilityComplaintsData = (facilityComplaints) => {
...
-    status: complaint.status
+    status: complaint.status,
+    allegations: getComplaintAllegations(complaint.allegations)
   })
   )
 }
 
+const getComplaintAllegations = (allegations) => {
+  return allegations.map((allegation, index) => ({
+    index_subcomponent: `${index + 1}.`,
```
Comment (Nasir2712) [inline app/javascript/selectors/facilityDataSelectors.js]: why is this being used?

Comment (sunilb0575) [reply app/javascript/selectors/facilityDataSelectors.js]: To show number for each row.

**Pull request merged**